12-watchtower-API.md

```diff
@@ -270,73 +363,31 @@ Sample code (python) for the client to prepare the `encrypted_blob`:
-
-[ FIXME: define signature serialization formats]
-
-### Range of values 
-
-The ```watchtower_signature_algorithm``` can be ECDSA or SCHNORR. The other new fields are self-explanatory. 
-
-### Rationale
-
-We assume the reader has a well-known public key for the WatchTower. It is outside the scope of this BOLT to dictate how the public key is retrieved and associated with the WatchTowers identity. 
-
-The signed receipt includes two new groupings: 
-- **Verifying WatchTower signature**: Both ```watchtower_signature_algorithm``` and ```watchtower_signature``` informs the reader how to verify the receipt's signature. 
-- **Conditional transfer**: Both ```payment_hash``` and ```payment_secret``` can be used to provide a fair exchange of the signed receipt and WatchTower payment over the lightning network. If the WatchTower is willing to accept a job without a new payment, then it will return the ```payment_secret``` immediately. 
-
+	    
 ## Payment modes 
 
-It is outside the scope of this BIP to dictate how a customer will pay the WatchTower. Generally, there are three approaches: 
+Although this BOLT does not enforce any specific payment method to be adopted, it is worth mentioning the three most common ones:
 
 **On-chain bounty**. An additional output is created in the justice transaction that will reward the WatchTower. 
 
-**Micropayments**. A small payment is sent to the WatchTower for every new job (i.e. over the lightning network)
-
-**Subscription**. WatchTower is periodically rewarded / paid for their service to the customer. (i.e. over the lightning network or dirty fiat subscription). 
+**Micropayments**. A small payment is sent to the WatchTower for every new job (e.g. over the lightning network)
 
-Both micropayments and subscriptions are favourable for a WatchTower.
+**Subscription**. WatchTower is periodically rewarded / paid for their service to the customer. (e.g. over the lightning network or fiat subscription). 
 
-**Problems with onchain bounty** We highlight that the on-chain bounty approach is not ideal for a watching network. It lets the customer hire N WatchTowers (O(N) storage for each tower) and only one WatchTower will be rewarded upon collecting the bounty. If this approach was sought, then the miners may end up becoming WatchTowers as they can very easily front-run an entire watching network. Thus to reduce the power and responsibility of miners, we should avoid this approach and advocate that miners do not participate in a watching network. 
- 
-## Number of updates
+Both micropayments and subscriptions are favourable for a WatchTower. The on-chain bounty approach is not ideal for a watching network, it lets the customer hire N WatchTowers (O(N) storage for each tower) and only one WatchTower will be rewarded upon collecting the bounty. On top of that, the onchain bounty allows a network-wise DoS attack for free.
```

Comment:
As an algorithmist, I dislike reusing `N` for two unrelated variables (that is: number of watchtowers, number of transactions in the channel). I 'd just say "hire many watchtowers" without explicitly naming the variable, since it's not used anyway. O(N) for the number of transactions is probably standard in payment channels literature, so I'd keep this one.